README.md

```diff
@@ -250,10 +250,11 @@ consensus and community norms, not impose more structure than necessary.
 ## License
 [License]: #license
 
-All repositories under the amethyst/ organization are Apache 2.0 or MIT licensed.
+With the exception of amethyst/website, all repositories under the amethyst/ organization are Apache 2.0 or MIT licensed. Amethyst/website is licensed under the CC BY-SA 4.0 license.
```

Comment:
I don't really think that's the best way to go. Why do you want to say that in the first place?

Reply:
Because it was in the Rust RFC README, from which I shamelessly ripped this off. Why don't you think this is the best way to go?

Reply:
Because if we add new repos that have exotic licences, we would have to maintain this arbitrary list as well.

Reply:
Should we be adding new repos that have exotic licenses without discussion? That's generally considered to be a bad idea, as it can taint the entire project. Choosing to allow additional licenses should be a Big Deal, imo. There's even repo scanners to look for toxic licenses hidden in projects.

I'm ok with removing this, but I'd like @torkleyy and @LucioFranco to weigh in. An explicit statement about what licenses we use in some fashion is, I think, a good thing.

Reply:
I want to keep everything consistent so I am for keeping this.

Reply:
@amethyst/core @TimonPost @magnonellie anyone else want to weigh in?

Reply:
I personally think that it's pretty bad that the website is licensed under CC as none of the CC licenses are really meant for code. I think that we should relicense it to Apache 2.0 or MIT for **code** and then CC BY for all **assets** which I think would be a pretty good pattern to keep throughout all our projects. The difference between us and Rust is that we are, to some extent, dealing with assets and possibly even work that we don't own the rights for. Maybe changing the text to something like this could work:

```suggestion
All code in repositories under the amethyst/ organization is Apache 2.0 or MIT licensed while all assets are Creative Commons Attribution 4.0 licensed if not otherwise stated.
```

 I chose CC BY instead of BY SA because SA would prevent people from making a commercial game that has our logo in the splash screen for example :sweat_smile:

Reply:
Okay I think that's pretty good. Note that the current logo is CC BY SA licensed by ebkalderon.